sample.env

```diff
@@ -13,6 +13,7 @@ REACT_APP_UNCHAINED_COSMOS_WS_URL=wss://dev-api.cosmos.shapeshift.com
 REACT_APP_PORTIS_DAPP_ID=8609e6a8-e0dc-45e6-a0ad-edde63a4cdda
 REACT_APP_ETHEREUM_NODE_URL=https://mainnet.infura.io/v3/d734c7eebcdf400185d7eb67322a7e57
 REACT_APP_KEEPKEY_VERSIONS_URL=https://static.shapeshift.com/firmware/releases.json
+REACT_APP_WALLET_MIGRATION_URL=http://localhost:4000/api/migrate
```

Comment:
can you provide the URL for what to point hosted environments at please?

Reply:
I'd be happy to do that, we have a staging instance up. I'm just wary of how much we want to obfuscate the endpoint api.

I know a user could just open their network tab and get it, but I was erring on the side of caution.

Reply:
everything is client side/public and should be treated accordingly. security through obscurity doesn't work :)